.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+
+language: node_js
+node_js:
```

Comment:
is this from our tooling? we should be testing on node 10

Reply:
it's from tooling, yes.

Reply:
cc @jquense